Resources/Prototypes/Loadouts/Jobs/Civilian/chef.yml

```diff
@@ -8,6 +17,19 @@
   equipment:
     head: ClothingHeadHatChef
 
+- type: loadout
+  id: ChefHeadHat
+  equipment: ChefHeadHat
+
+- type: startingGear
+  id: ChefHeadHat
+  equipment:
+    head: ClothingHeadHatHeadChef 
+  effects:
+  - !type:GroupLoadoutEffect
+    proto: HeadChef
```

Comment:
excellent highlighting github, very cool

Reply:
Thanks guys. This PR in particular I had a quite bad workflow where I was writing code in notepad and playtesting at the same time and then manually copying those changes into github from notepad. I missed this (and a couple other things I since found) doing that.